src/outputs/alsa.c

```diff
@@ -1244,6 +1337,9 @@ alsa_init(void)
   if (type && (strcasecmp(type, "alsa") != 0))
     return -1;
 
+  if (cfg_getbool(cfg_audio, "list_devices"))
```

Comment:
Yes, I agree that this doesn't require an option. We could also print it with normal log level, it is just a few lines, right?

Reply:
Yes, its not a lot of output, just one line for each ALSA sound card - on my RPI2 with the default card + a DAC hat:

```
[ INFO]   laudio: Available ALSA playback/mixer(s) on 'hw:0' (bcm2835 ALSA): 'PCM' 
[ INFO]   laudio: Available ALSA playback/mixer(s) on 'hw:1' (IQaudIODAC): 'Analogue' 'Digital'
```
I downgraded this to `INFO` to keep  this log line in sync with the other server logs for the alsa during setup - I'll put it to `LOG` if you feel this way:
```
[  LOG]     mdns: Avahi state change: Client running
[ INFO]     main: Initializing database
[  LOG]       db: Configured to use database file '/var/cache/forked-daapd/songs3.db'
[  LOG]       db: Now vacuuming database, this may take some time...
[  LOG]       db: Database OK with 132 active files and 24 active playlists
[ INFO]    cache: cache thread init
[ INFO]   laudio: Available ALSA playback/mixer(s) on 'hw:0' (HDA Intel): 'Master' 'Headphone' 'Speaker' 'PCM' 'Mic' 'Beep' 
[ INFO]   laudio: Adding ALSA device 'default' with name 'default'
[ INFO]   laudio: Adding ALSA device 'equal' with name 'equalisd device'
[ INFO]   laudio: Adding ALSA device 'dac' with name 'dac'
[ INFO]   laudio: Adding ALSA device 'hw:0' with name 'headphone'
```